workflows/genomeassembly.nf

```diff
@@ -91,13 +91,13 @@ workflow GENOMEASSEMBLY {
     //
     // LOGIC: SEPARATE READS PATHS INTO A DIFFERENT CHANNEL
     //    
-    PREPARE_INPUT.out.hic.map{ meta, reads, motif -> reads }.set{ hic_reads_ch }
+    PREPARE_INPUT.out.hic.map{ meta, reads, motif, hic_aligner -> reads }.set{ hic_reads_ch }
 
     //
     // SUBWORKFLOW: GENERATE KMER DATABASE AND PROFILE MODEL
     //
-    GENOMESCOPE_MODEL( hifi_reads_ch )   
-    ch_versions = ch_versions.mix(GENOMESCOPE_MODEL.out.versions)
+    /*GENOMESCOPE_MODEL( hifi_reads_ch )   
+    ch_versions = ch_versions.mix(GENOMESCOPE_MODEL.out.versions)*/
```

Comment:
Was it commented out with the purpose of debugging and does it need uncommenting?

Reply:
yes, that was the purpose :) I will remove them now.